edx_notifications/server/web/static/edx_notifications/js/views/counter_icon_view.js

```diff
@@ -0,0 +1,66 @@
+var CounterIconView = Backbone.View.extend({
```

Comment:
Jasmine test harnessing (and initial sample test) can be found at: https://github.com/edx/edx-notifications/pull/15

We're waiting to merge that in after we merge in the RequireJS changes (https://github.com/edx/edx-notifications/pull/16), as we'll have to update the Jasmine harnessing to account for RequireJS